homeassistant/components/tellduslive/__init__.py

```diff
@@ -73,7 +75,8 @@
 
     interval = timedelta(seconds=entry.data[KEY_SCAN_INTERVAL])
     _LOGGER.debug('Update interval %s', interval)
-    async_track_time_interval(hass, client.update, interval)
+    hass.data[INTERVAL_TRACKER] = async_track_time_interval(
```

Comment:
Would it be better to store this at the client?

Reply:
Maybe, you decide.